packages/makeswift/src/errors/MakeswiftError.ts

```diff
@@ -3,7 +3,7 @@
  * where we want the program to simply exit.
  */
 export default class MakeswiftError extends Error {
-  constructor(message: string) {
+  constructor(message: string | undefined = '') {
```

Comment:
Should we make the default message something more descriptive? Perhaps "An unexpected error occurred"?

Reply:
I think I'm actually going to make this a required string. There is only 1 use case that I've had for an empty string, and this is making me realize I should've just done `process.exit(0)`

Reply:
Yeah, that makes sense